widgets/images/imageList.widget.js

```diff
@@ -45,11 +45,11 @@ class myWidget extends ListWidget {
 
     if (images) {
       images.forEach((image) => {
-        const getTag = (tag, part) => tag ? tag[0].split(':')[part] : 'none'
+        const getTag = (tag, part) => (tag && tag.length) ? tag[0].split(':')[part] : 'none'
 
         imageList.push([
           image.Id.substring(7, 12),
-          image.RepoDigests ? image.RepoDigests[0].split('@')[0] : getTag(image[2], 0),
+          (image.RepoDigests && image.RepoDigests.length) ? image.RepoDigests[0].split('@')[0] : getTag(image[2], 0),
```

Comment:
I would prefer to use the concatenation operator to check the values, but the linter prevents the use of this operator.

`image.RepoDigests && image.RepoDigests.length` vs. `image.RepoDigests?.length`

The dependency "default": "^12.0.1" would need to be updated, the current version is 17.1.0 and the operator is supported from version 15. But this leads to a whole list of adjustments that would then have to be made. I am not sure how to handle this?

Reply:
It's ok to use as-is and skip the optional chaining due to support of older Node.js versions.